docs/user-documentation/resource-nodes.md

```diff
@@ -0,0 +1,137 @@
+solution for modeling our resource nodes and media, compatibility with third-party modules is virtually guaranteed.
+This opens up a plethora of solutions from the Drupal community that will save you untold time
+and effort when implementing your repository with Islandora.
+
+## Properties
+
+Resource nodes have some basic properties that are common to all nodes, regardless 
+of its content type.  These properties are not fields.  This means that their settings cannot be
+changed and they cannot be removed. Their name, what type of data they hold, etc... are all baked in.
+Here's an example of the basic properties you'll see on a node:
+
+```
+nid: 1
+uid: 1
+title: "I am an Islandora 8 object"
+created: 1550703004
+changed: 1550703512
+uuid: 02932f2c-e4c2-4b7e-95e1-4aceab78c638
+type: islandora_object
+status: 1
+```
+
+As you can see, it's all system data used at the Drupal level to track the basics.
+
+Property | Value
+------------ | -------------
+nid | The local ID for the node
+uuid | The global ID for any entity
+title | The title for the node
+created | Timestamp of when the node was created
+changed | Timestamp of when the node was last updated
+type | Content type (e.g. which group of fields are present on the node)
+status | Published, unpublished, etc...
+
+!!! note "Compared to Islandora 7"
+    These node properties directly correspond to following Islandora 7.x object properties:
+
+    Islandora 7 | Islandora 8
+    ----------- | -----------
+    owner | uid
+    dc.title | title
+    PID | uuid
+    status | status
+
+## Fields
+
+In addition to the basic node properties identified above, resource nodes (like all Drupal nodes) can have fields.
+Most of what we would think of as descriptive metadata is stored as fields. Resource nodes use 'content types' to define a specific set of required and optional
+fields it has; we can think of content types as metadata profiles for our objects.
+For example, you might have a content type for a set of repository objects that have very specialized metadata requirements but
+another content type for generic repository objects that share a more general set of metadata fields.
+A resource node's content type is set on its creation and is immutable.
+The section on [metadata](metadata.md) describes in more detail how fields on Islandora objects work.
+
+Islandora has a notion of a _content model_, which is used to identify what type of content is
+being represented by a node (e.g. an image, a video, a collection of other items, etc...). This is done
+using a special field, _Model_, which accepts taxonomy terms from the _Islandora Models_ vocabulary.
+By applying a term from the Islandora Models vocabulary to a node, Islandora will become aware
+of how to handle the node in response to certain events, like choosing a viewer or generating derivatives.
+
+![Model tags](../assets/resource_nodes_model_tags.png)
+
+!!! note "Compared to Islandora 7"
```

Comment:
This is more of a meta question about "Compared to Islandora 7" notes... 
"!!! note  "Compared to Islandora 7" is the convention that's being used throughout the Islandora 8 documentation?

Reply:
🤷‍♂️ That note already existed, just in a different spot. I think @manez is a better person to ask about that.

Reply:
If I recall correctly, we used the "compared to Islandora 7" notes as a compromise to give useful info to 7.x users, without making out 8 docs all about 7. Thus, they're broken out as special notes.